wifiphisher.py

```diff
@@ -58,6 +57,7 @@ def parse_args():
     parser.add_argument("-p", "--packets", help="Choose the number of packets to send in each deauth burst. Default value is 1; 1 packet to the client and 1 packet to the AP. Send 2 deauth packets to the client and 2 deauth packets to the AP: -p 2")
     parser.add_argument("-d", "--directedonly", help="Skip the deauthentication packets to the broadcast address of the access points and only send them to client/AP pairs", action='store_true')
     parser.add_argument("-a", "--accesspoint", help="Enter the MAC address of a specific access point to target")
+    parser.add_argument("-s", "--style", help="Specify a style for the fake configuration page. Options: apple, cisco, dlink, dlinksecurespot, netgear.", default="minimal")
```

Comment:
The '-s' option already exists for the --skip argument. Please use a different one.